docs/application-development/testing.md

```diff
@@ -111,7 +113,23 @@ Content is currently unavailable.
 </TabItem>
 <TabItem value="python">
 
-Content is currently unavailable.
+To test a Heartbeat in an Activity, you can use the `on_heartbeat()` property of the `ActivityEnvironment` class. This property allows you to set a custom function that will be called every time the `activity.heartbeat()` function is called within the Activity.
+
+```python
+async def test_heartbeats():
+    async def activity_with_heartbeats(param: str):
```

Comment:
May want to move this outside and add a `@activity.defn`. Not required but it can be clearer for docs.